### DISCUSSION TITLE
posts/React/%5BReact%20DeepDive%5D%20React%EC%9D%98%20%EA%B0%80%EC%83%81%20DOM

### DISCUSSION THREAD

**Discussion by giscus** [phnml1/my-blog#11] [General]
# posts/React/%5BReact%20DeepDive%5D%20React%EC%9D%98%20%EA%B0%80%EC%83%81%20DOM

리액트 딥다이브 책 내용 중 '가상 DOM과 파이버'의 정리 내용입니다.

https://phnml1.github.io/posts/React/%5BReact%20DeepDive%5D%20React%EC%9D%98%20%EA%B0%80%EC%83%81%20DOM

**Comment by phnml1** [comment]
댓글 테스트
